java/src/com/ibm/streamsx/rest/Operator.java

```diff
@@ -50,13 +51,27 @@
     @Expose
     private String self;
 
-    /**
-     * this function is not intended for external consumption
-     */
-    void setConnection(final StreamsConnection sc) {
+    private void setConnection(final StreamsConnection sc) {
         connection = sc;
     }
 
+    static final List<Operator> getOperatorList(StreamsConnection sc, String operatorsList) {
+        List<Operator> opList;
+        OperatorArray opArray;
+        try {
+            opArray = new GsonBuilder().excludeFieldsWithoutExposeAnnotation().create().fromJson(operatorsList,
+                    OperatorArray.class);
+
+            opList = opArray.operators;
+            for (Operator op : opList) {
+                op.setConnection(sc);
+            }
+        } catch (IllegalStateException e) {
+            opList = new ArrayList<Operator>();
```

Comment:
It just seems strange to hide errors, and to have extra code that is not really needed.